seqr/views/utils/file_utils_tests.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
you should rename this file to _2_3_tests.py so it will get tested in python 3 going froward

Reply:
ok